<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWallet</class>
 <widget class="QDialog" name="NewWallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="brainWallet">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>compatible to www.brainwallet.org - a simple sha256 hash</string>
   </property>
   <property name="text">
    <string>Brain Wallet</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="importElectrum">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Electrum Seed</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="newElectrum">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New Electrum Seed</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="seed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter a seed or Pass Phrase</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewWallet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewWallet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
